### DISCUSSION TITLE
[SFC] Star Ocean

### DISCUSSION THREAD

**Discussion by PsyK0p4T** [sanni/cartreader#563] [Need help]
Recently redumped a large amount of JPN carts, and among them Star Ocean (SHVC-ARFJ-JPN)
This cart is the only one giving me strange behaviors with the reader.
By looking at the PCB, the two chips are 32+16Mbits so 6MB as expected rom size.

Detection looks correct : 
![image](https://user-images.githubusercontent.com/87064902/195717572-b0ea2a34-de01-4723-a109-6a126c46658c.png)

After launching the dump process, progress bar is acting strange. Ending the line and restarting an other one, endless.
![image](https://user-images.githubusercontent.com/87064902/195717749-cc1d7dda-2f36-4b2d-93b7-d031ced2d71a.png)
![image](https://user-images.githubusercontent.com/87064902/195717858-c62105ae-1e6b-449e-aa5f-95532d8784d1.png)

If the process is aborted too early, I just have a 0b file on SD.
If I let it run for really longer, and then abort, I have a 12mb file. 
Trimmed it to the first 6MB, and the rom obtained was matching the expected "good dump" hashes.

I was using HW3 and V9.4 at this time. Unfortunately I had to send the cartridge back, so won't be able to do more for now.
More tests from people with the same game could be good...

**Comment by sanni** [comment]
snes.txt database error, banks should be 96 not 192. So it will over-dump to twice the size right now. 
The progress bar behavior is intended, although it only should use 6 bars if I understand the code correctly, one for every SDD1 memory map. Because of the database error it tries to dump 12 memory maps.

**Comment by PsyK0p4T** [comment]
Oh. Good find. I'll try to get my hands back on the cart to test it again. Thanks for the fix !

**Comment by PsyK0p4T** [comment]
Just a quick update to this thread : the banks edit worked !
I just tested the cartridge again and got the correct rom size and crc.
Also confirmed that 6 progress bars are displayed on screen during the process.
